lib/tapioca/gem/pipeline.rb

```diff
@@ -124,7 +130,7 @@ def dispatch(event)
         case event
         when Gem::SymbolFound
           on_symbol(event)
-        when Gem::ConstantFound
+        when Gem::ConstantFound, Gem::NotOwnedConstantFound
```

Comment:
If `NotOwnedConstantFound` subclasses `ConstantFound`, we don't need this addition.